<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_Top" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="Layout_Top">
         <item>
          <widget class="QToolButton" name="Button_NewTab">
           <property name="text">
            <string>New Tab</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_Clear"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="chat_tabs">
      <property name="styleSheet">
       <string notr="true">QTabWidget#Modified &gt; QTabBar::tab:top, QTabWidget#Modified &gt; QTabBar::tab:bottom {
    background: #bababa;
    border: 1px solid #676a6c;
    border-radius: 5px;
    height: 23px;
    min-width: 90px;
    padding: 2px;
}

QTabWidget#Modified  &gt; QTabBar::tab::title {
    font-weight:bold;
    font-family:Trebuchet MS;
    font-size: 14px;
}

QTabWidget#Modified &gt; QTabBar::tab:hover{
    border-color: #077cff;
    color: white;
    background: #88bcf1;
}

QTabWidget#Modified &gt; QTabBar::tab:selected {
    border-color: #077cff;
    color: white;
    background: #66ace1;
}

QTabWidget#Modified &gt; QTabBar::tab:top:selected , QTabWidget#Modified &gt; QTabBar::tab:bottom:selected {
    /* expand/overlap to the left and right by 4px */
    margin-left: -2px;
    margin-right: -2px;
}

QTabWidget#Modified &gt; QTabBar::tab:top:first,QTabWidget#Modified &gt; QTabBar::tab:bottom:first {
    margin-left: 15px;
}

QTabWidget#Modified &gt; QTabBar::tab:top:last, QTabWidget#Modified &gt; QTabBar::tab:bottom:last {
    margin-right: 0px;
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
